helm/spark-yotpo/values.yaml

```diff
@@ -34,6 +34,11 @@ sparkConf:
   spark.serializer: org.apache.spark.serializer.KryoSerializer
   spark.sql.hive.convertMetastoreParquet: false
   spark.sql.legacy.allowCreatingManagedTableUsingNonemptyLocation: true
+historyServer:
+  enabled: false
+  conf:
+    spark.eventLog.enabled: true
+    spark.eventLog.dir: s3://event-log-bucket
```

Comment:
@lyogev this cant be env var, spark dont know how to extrapolate it during runtime...what do u thin we should do? i think we should remove this code from here and place in a closed source proejct, WDYT?